conf/modules.config

```diff
@@ -157,5 +157,30 @@ process {
         ]
         ext.prefix = { "${report}" }
     }
+
+    withName: AMRFINDERPLUS_UPDATE {
+        publishDir = [
+            path: { "${params.outdir}/amrfinderplus/db" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: AMRFINDERPLUS_RUN {
+        publishDir = [
+            path: { "${params.outdir}/amrfinderplus" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: HAMRONIZATION_AMRFINDERPLUS {
+        publishDir = [
+            path: { "${params.outdir}/hamronization/amrfinderplus" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = { "${report}" }
```

Comment:
Where does `report` come from here?

Reply:
Actually, I left it there, because it appeared in all hamronization modules ...